data/src/main/java/com/teamagam/gimelgimel/data/base/repository/SingleReplayRepository.java

```diff
@@ -3,12 +3,12 @@
 import rx.Observable;
 import rx.subjects.PublishSubject;
 
-public class SingleValueRepository<T> {
+public class SingleReplayRepository<T> {
```

Comment:
I think you should delete one of the repositories.
If we want replay of 2 items? We won't create new class of "DoubleReplayRepository".

More generic class will be nice here.